packages/core/utils/__tests__/makeComputedGetters.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { ref } from '@vue/composition-api';
+import { makeComputedGetters } from '../src/makeComputedGetters';
+
+describe('[CORE core] makeComputedGetters', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should map all getter functions to computed getters', () => {
+    const mockGetters = {
+      getProducts: param => `${param}`,
+      getTree: param => `${param}`
+    };
+
+    const computedGetters = makeComputedGetters(mockGetters);
+
+    expect(computedGetters.getProducts('test')).toEqual(ref('test'));
+    expect(computedGetters.getTree('test')).toEqual(ref('test'));
```

Comment:
shouldn't be read-only computed?

Reply:
For testing proposes, I used ref to check wheater the computed properties returned the reactive value.